### PR TITLE
Cguard90

### PR DESCRIPTION
**Pull request by cguard90**
Hey Pete & Carolina, The logic works to have the questions cycle through to each other. At the end it currently cycles back to the home page as we have no stats calculations yet. I'm wondering if we should prioritize the stats calculation, which is just an algorithm, or dive into the AJAX parts, which is a skill we will need to utilize on the assessment. Let me know your thoughts.

I'd like to add the AJAX on at least 2 pages, so if you could please let me know which 2 I can do I would appreciate that. 

Last note, if you post to github, i get emailed the notifications. If you prefer to text / call (don't call, i don't answer phone calls lol), my number is 716-628-7587, but all emails go to my phone as well.

TLDR version: here is the code. what should we prioritize next? I will keep in touch. Go Team!

### CODE REVIEW

**Comment by ghost** [comment]
@caromedellin @cguard90 
I'd like to focus on AJAX. That means we should only have one of us working on the application.js at once (or two of us in a pair, somehow).

Having looked over my code review from Thursday, a recurring theme was that a controller should only render (the erb method) a view that belongs to that controller: questions_controller only renders '/questions...' views, etc. At the moment, some of our controllers aren't following this rule, so I'd be happy to rectify that while you get started with AJAX, Chris.

I don't remember whether you were going to come in today, Carolina, so if you are, we can pair on whatever other things need doing.

**Comment by ghost** [comment]
@cguard90 @caromedellin 

Also, I'll go ahead and merge this pull request.

**Comment by cguard90** [comment]
```
                                                                                  Thanks Pete, I agree about the controllers, and I was thinking that as I wrote the logic this morning, Carolina will be in in about an hour                                                                                                                                                                                                                                                                                                                                         Sent from my BlackBerry 10 smartphone on the Verizon Wireless 4G LTE network.                                                                                                                                                                                                                From: Peter WoodSent: Sunday, January 10, 2016 11:06 AMTo: cguard90/zoosReply To: cguard90/zoosCc: Chris GuardSubject: Re: [zoos] Cguard90 (#30)@cguard90 @caromedellin 
```

Also, I'll go ahead and merge this pull request.

—Reply to this email directly or view it on GitHub.
